db/migrate/20210407081304_create_legal_framework_merits_task_list.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLegalFrameworkMeritsTaskList < ActiveRecord::Migration[6.1]
+  def change
+    create_table :legal_framework_merits_task_lists, id: :uuid do |t|
+      t.references :legal_aid_application, foreign_key: true, type: :uuid, index: { name: 'idx_lfa_merits_task_lists_on_legal_aid_application_id' }
+      t.string :serialized_data
+      t.timestamps
```

Comment:
Ideally, we would want to make the serialized data `text` rather than `string` because it could potentially be very big, especially when we start implementing other endpoints on this API.  On postgres in fact, there is no difference, but on other databases there are, so in the interests of being database agnostic ....